Content.Client/PDA/PdaMenu.xaml

```diff
@@ -28,20 +28,24 @@
                       HorizontalExpand="True"
                       MinSize="50 50"
                       Margin="8">
-            <RichTextLabel Name="PdaOwnerLabel" Access="Public" />
-            <RichTextLabel Name="IdInfoLabel"
-                           Access="Public"
-                           HorizontalExpand="True" />
-            <PanelContainer>
-                <BoxContainer Orientation="Horizontal">
-                    <RichTextLabel Name="StationNameLabel"
-                                   Access="Public"
-                                   HorizontalExpand="True" />
-                </BoxContainer>
-            </PanelContainer>
-            <RichTextLabel Name="StationAlertLevelLabel" Access="Public" />
-            <RichTextLabel Name="StationTimeLabel" Access="Public" />
-            <RichTextLabel Name="StationAlertLevelInstructions" Access="Public" />
+            <ContainerButton Name = "PdaOwnerButton" Access="Public">
+                <RichTextLabel Name="PdaOwnerLabel" Access="Public" />
+            </ContainerButton>
+            <ContainerButton Name = "IdInfoButton" Access="Public">
+                <RichTextLabel Name="IdInfoLabel" Access="Public"/>
+            </ContainerButton>
+            <ContainerButton Name = "StationNameButton" Access="Public">
+                <RichTextLabel Name="StationNameLabel" Access="Public"/>
+            </ContainerButton>
+            <ContainerButton Name="StationAlertLevelButton" Access="Public">
+                <RichTextLabel Name="StationAlertLevelLabel" Access="Public" />
+            </ContainerButton>
+            <ContainerButton Name = "StationTimeButton" Access="Public">
```

Comment:
The extraneous spaces here should probably be removed for cleaner XAML

```suggestion
            <ContainerButton Name="PdaOwnerButton" Access="Public">
                <RichTextLabel Name="PdaOwnerLabel" Access="Public" />
            </ContainerButton>
            <ContainerButton Name="IdInfoButton" Access="Public">
                <RichTextLabel Name="IdInfoLabel" Access="Public"/>
            </ContainerButton>
            <ContainerButton Name="StationNameButton" Access="Public">
                <RichTextLabel Name="StationNameLabel" Access="Public"/>
            </ContainerButton>
            <ContainerButton Name="StationAlertLevelButton" Access="Public">
                <RichTextLabel Name="StationAlertLevelLabel" Access="Public" />
            </ContainerButton>
            <ContainerButton Name="StationTimeButton" Access="Public">
```

Reply:
Also public access for the button seems to be useless here(?)

Reply:
Fix public access, fix spaces beetwen ids